tools/tools/atsectl/atsectl.c

```diff
@@ -291,9 +291,10 @@ int
 main(int argc, char **argv)
 {
 	char ch, *s;
```

Comment:
Could just change `ch` to be an int instead?

Reply:
Yeah, that's the right fix, and we can probably direct commit that upstream.

Reply:
Apologies, did not get my github notifications right and completely forgot about that. Did change it to an int.